lib/Backend/SwitchIRBuilder.cpp

```diff
@@ -2,8 +2,8 @@
 // Copyright (C) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
-#include "BackEnd.h"
-#include "DataStructures\QuickSort.h"
+#include "Backend.h"
+#include "DataStructures/quicksort.h"
```

Comment:
> Is quicksort.h eventually going to be renamed to QuickSort.h?

A few lower case hearders that looks well established and I didn't bother to rename. I can rename it if you prefer.